angio-frontend/src/modules/common/views/Landing.vue

```diff
@@ -0,0 +1,127 @@
+                            </v-card-title>
+                            <v-card-text class="headline font-weight-light pt-0">{{ $t('common.view.landing.content.main.content') }}</v-card-text>
+                        </v-flex>
+
+                        <v-flex xs12>
+                            <v-card-title primary-title>
+                                <h4 class="display-1 font-weight-regular mb-0">{{ $t('common.view.landing.content.analyses.title') }}</h4>
+                            </v-card-title>
+                            <v-card-text
+                                    class="headline font-weight-light pt-0 pr-0 pb-0"
+                                    v-for="analyse in $t('common.view.landing.content.analyses.content')"
+                            >
+                                &#9679; {{ analyse }}
+                            </v-card-text>
+                        </v-flex>
+
+                        <v-flex xs12>
+                            <v-card-title primary-title>
+                                <h4 class="display-1 font-weight-regular mb-0">{{ $t('common.view.landing.content.abilities.title') }}</h4>
+                            </v-card-title>
+                            <v-card-text
+                                    class="headline font-weight-light pt-0 pr-0 pb-0"
+                                    v-for="ability in $t('common.view.landing.content.abilities.content')"
+                            >
+                                &#9679; {{ ability }}
+                            </v-card-text>
+                        </v-flex>
+
+                        <v-flex xs12>
+                            <v-card-title primary-title>
+                                <h4 class="display-1 font-weight-regular mb-0">{{ $t('common.view.landing.content.parameters.title') }}</h4>
+                            </v-card-title>
+                            <v-card-text
+                                    class="headline font-weight-light pt-0 pr-0 pb-0"
+                                    v-for="(parameter, index) in $t('common.view.landing.content.parameters.content')"
+                            >
+                                {{ incrementIndex(index) }}. {{ parameter }}
+                            </v-card-text>
+                        </v-flex>
+
+                        <v-flex xs12>
+                            <v-card-title primary-title>
+                                <h4 class="display-1 font-weight-regular mb-0">{{ $t('common.view.landing.content.work.title') }}</h4>
+                            </v-card-title>
+                            <v-card-text
+                                    class="headline font-weight-light pt-0 pr-0 pb-0"
+                                    v-for="(step, index) in $t('common.view.landing.content.work.content')"
+                                    v-if="index === 0"
```

Comment:
Using v-if together v-for is not recommended by doc [1](https://ru.vuejs.org/v2/guide/conditional.html#v-if-%D0%B2%D0%BC%D0%B5%D1%81%D1%82%D0%B5-%D1%81-v-for) and [2](https://ru.vuejs.org/v2/guide/list.html#%D0%9E%D1%82%D0%BE%D0%B1%D1%80%D0%B0%D0%B6%D0%B5%D0%BD%D0%B8%D0%B5-%D0%BE%D1%82%D1%84%D0%B8%D0%BB%D1%8C%D1%82%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%BD%D1%8B%D1%85-%D0%BE%D1%82%D1%81%D0%BE%D1%80%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%BD%D1%8B%D1%85-%D1%80%D0%B5%D0%B7%D1%83%D0%BB%D1%8C%D1%82%D0%B0%D1%82%D0%BE%D0%B2)

Reply:
Done